Today/Controllers/ReminderListViewController/ReminderListViewController.swift

```diff
@@ -49,7 +49,7 @@ class ReminderListViewController: UITableViewController {
     private func addReminder() {
         let storyboard = UIStoryboard(name: Self.mainStoryboardName, bundle: nil)
         let detailViewController: ReminderDetailViewController = storyboard.instantiateViewController(identifier: Self.detailViewControllerIdentifier)
```

Comment:
We have a special extension method of UIViewController to initialize ViewControllers. Use that one.

Reply:
Okay trying to implement it. Resolving some errors regarding it.